src/interfaces/AggregatorV3Interface.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity >=0.5.0;
 
 /// @dev From
-/// https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol.
+/// https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol.
```

Comment:
```suggestion
/// https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol
```

to make it easier to copy paste maybe

Reply:
```suggestion
/// https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol
```

I don't really care whether we have a dot at the end
There's something that bothers me more: why are we based on the `develop` branch?

Reply:
And why v0.6 instead of v0.8

Reply:
I just used the link suggested in the issue, but I agree we should use this one `https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol`